### PR TITLE
Add fq 0.0.4

### LINKED ISSUE #3294: Add fq to scoop
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [x] a non-GUI tool
- [x] a reasonably well-known and widely used developer tool
- [x ] the latest stable version of the program
- [x] the full version i.e. not a trial version
- [x] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** fq

**Description:** A CLI Tool, language and decoder for working with binary data, useful for analysis.

**Homepage:** https://github.com/wader/fq

**Download link(s):** https://github.com/wader/fq/releases/download/v0.0.4/fq_0.0.4_windows_amd64.zip

**Some indication of popularity/repute:** Github 3.3k stars

I have a branch awaiting a PR that adds this to main if you wish to continue.

https://github.com/thushan/scoop-main/blob/add-fq/bucket/fq.json

### PR DESCRIPTION
**Pull request by thushan**
Closes #3294

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/fq.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.0.4",
+    "description": "Tool, language and decoders for working with binary data.",
+    "homepage": "https://github.com/wader/fq",
+    "license": "MIT",
```
Comment (rashil2000) [inline bucket/fq.json:5]: Source?

Comment (thushan) [reply bucket/fq.json:5]: for license?

Comment (thushan) [reply bucket/fq.json:5]: They should update the repo to make it clearer though I agree.

Comment (rashil2000) [reply bucket/fq.json:5]: There is no mention of MIT in that file. Add a field called "url" in the license object, and link that file there.

File: bucket/fq.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.0.4",
+    "description": "Tool, language and decoders for working with binary data.",
+    "homepage": "https://github.com/wader/fq",
+    "license": {
+        "url": "https://github.com/wader/fq/blob/master/LICENSE"
```
Comment (rashil2000) [inline bucket/fq.json:6]:
```suggestion
    "license": {
        "identifier": "Unknown",
        "url": "https://github.com/wader/fq/blob/master/LICENSE"
```

Looks like we need this field too to pass the CI

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
I think "Unknown" would be better, as they don't mention it as Freeware either.

**Comment by thushan** [comment]
Agreed, I updated it as per your recommendation @rashil2000! I've not used the expanded License object yet on scoop, good to know!

**Comment by thushan** [comment]
Thanks much appreciated my good sir!

**Comment by rashil2000** [comment]
No worries :)

**Comment by wader** [comment]
Hi, more license details have been added now https://github.com/wader/fq/pull/138

**Pull request merged**